test/testlock.c

```diff
@@ -23,7 +23,7 @@
 static SDL_mutex *mutex = NULL;
 static SDL_threadID mainthread;
```

Comment:
While we're at it, what about `SDL_mutex`, `SDL_sem`, `SDL_threadID`, etc.?

Reply:
I don't know if there is a real convention about `SDL_Foo` vs `SDL_foo`, otherwise I'd be glad to uniform that.